build/root/Makefile

```diff
@@ -363,6 +363,21 @@ lint:
 	hack/verify-golangci-lint.sh
 endif
 
+define KUBE_API_LINTER_HELP_INFO
```

Comment:
This is based on the golangci-lint target above

Reply:
Is having a separate make target worth the duplication (config file, script)? Is that for the sake of convenience (different output format, faster execution)?

If it's execution time, then perhaps we can use the same configuration file and enable the API linter only if invoked via a command line option, which then could be added by a 'verify-kube-api-linter.sh` wrapper script which invokes `verify-golangci-lint.sh`.

Reply:
I had originally imagined we maybe wanted the linter as a separate verify job that only triggered when there were API changes, and so that if there was an issue with a false positive at some point, it could be overridden.

I think the conversation has moved to this should just be part of the standard verify step so perhaps there is less value in the separate makefile target

Personally, I'm using the `./hack/verify-kube-api-linter.sh` script directly to be able to run the linter specifically for finding and fixing/excluding the issues it's raising, I think it's useful being able to run this separately from the main linter for now at least

Reply:
Not statistical, but a quick comparison, the current suite of golangci-lint takes around 15s for me to run, where the KAL only script is about 5s, so there is a shorter feedback

If you think it's best to integrate directly into the `verify-golangci-lint.sh` file directly I'm happy to take direction on the best approach

Reply:
Ok, if that works I'll clean up all this extra stuff and merge the KAL config via a template into the main golangci-lint configuration

Reply:
> Not statistical, but a quick comparison, the current suite of golangci-lint takes around 15s for me to run, where the KAL only script is about 5s, so there is a shorter feedback

Note: in CI we will be starting from a clean slate without warm caches, when we're merging things that part is relevant.